src/v1/device_manager_client.ts

```diff
@@ -95,6 +95,10 @@ function isErrorStatusCode(statusCode: number): boolean {
   return false;
 }
 
+export interface IHash {
```

Comment:
Please change this to the following:
```
interface IHash {
    [key: string]: GetRegistryCredentialsResponse & {host: string}
}
```
This way we don't have to duplicate the JSON.parse logic